pkgs/tools/networking/ookla-speedtest/default.nix

```diff
@@ -13,6 +13,11 @@ let
       url = "https://install.speedtest.net/app/cli/${pname}-${version}-linux-aarch64.tgz";
       sha256 = "sha256-J2pAhz/hw8okohWAwvxkqpLtNY/8bbYHGhPQOo1DH9k=";
     };
+    x86_64-darwin = fetchurl {
+      url = "https://install.speedtest.net/app/cli/${pname}-${version}.84-macosx-x86_64.tgz";
+      sha256 = "sha256-FT925OUCortHDH98O0uK+qUOuYuxHoYhb8sai5JnbpQ=";
+    };
+    aarch64-darwin = x86_64-darwin;
```

Comment:
Ookla provides a single universal macOS binary.

Reply:
That doesn't seem right. How is a single universal binary even possible?

Reply:
@Kranzes that normals on macOS. Both binaries can be bundled into one.

![image](https://user-images.githubusercontent.com/6224096/170823555-6aff65eb-5807-4f9d-a6cb-9e148309bf1a.png)

Reply:
Interesting...

Reply:
```sh
$ file ./result/bin/speedtest
./result/bin/speedtest: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit executable x86_64Mach-O 64-bit executable x86_64] [arm64]
./result/bin/speedtest (for architecture x86_64):	Mach-O 64-bit executable x86_64
./result/bin/speedtest (for architecture arm64):	Mach-O 64-bit executable arm64
```